tasks/inventory_report/schemas/output.json

```diff
@@ -3,130 +3,114 @@
-                  "type": "array",
-                  "items": {
-                    "type": "object",
-                    "required": [
-                      "name",
-                      "cumulusArchiveLocation",
-                      "orcaArchiveLocation",
-                      "keyPath",
-                      "sizeBytes",
-                      "hash",
-                      "hashType",
-                      "version",
-                      "orcaEtag"
-                    ],
-                    "properties": {
-                      "filename": {
-                        "description": "The name and extension of the file.",
-                        "type": "string"
-                      },
-                      "cumulusArchiveLocation": {
-                        "description": "Cumulus bucket the file resides in.",
-                        "type": "string"
-                      },
-                      "orcaArchiveLocation": {
-                        "description": "ORCA S3 Glacier bucket the file resides in.",
-                        "type": "string"
-                      },
-                      "keyPath": {
-                        "description": "Full S3 path to the file including the file name and extension."
-                      },
-                      "sizeBytes": {
-                        "description": "Size in bytes of the file. From Cumulus ingest.",
-                        "type": "integer"
-                      },
-                      "hash": {
-                        "description": "Checksum hash of the file provided by Cumulus.",
-                        "type": ["string", "null"]
-                      },
-                      "hashType": {
-                        "description": "Hash type used to calculate the hash value of the file.",
-                        "type": ["string", "null"]
-                      },
-                      "version": {
-                        "description": "AWS provided version of the file.",
-                        "type": "string"
-                      },
-                      "orcaEtag": {
-                        "description": "AWS provided etag of the versioned file. Semi unique.",
-                        "type": "string"
-                      }
-                    }
-                  }
+                "hash": {
+                  "description": "Checksum hash of the file provided by Cumulus.",
+                  "type": [
+                    "string",
+                    "null"
+                  ]
+                },
+                "hashType": {
+                  "description": "Hash type used to calculate the hash value of the file.",
+                  "type": [
+                    "string",
+                    "null"
+                  ]
+                },
+                "version": {
+                  "description": "AWS provided version of the file.",

```

Comment:
May want to expand that this is the latest ORCA version of the file in the ORCA bucket provided by AWS.